src/main/java/com/github/hornta/race/objects/RaceSession.java

```diff
@@ -367,6 +364,65 @@ public void run() {
     });
   }
 
+  private List<RacePlayerSession> getStartOrderSessions() {
+    Map<UUID, RacePlayerStatistic> resultsByPlayerId = race.getResultByPlayerId();
+    List<RacePlayerSession> startOrderSessions = new ArrayList<>(playerSessions.values());
+    switch (race.getStartOrder()) {
+      case FASTEST:
+        Collections.sort(startOrderSessions, new Comparator<RacePlayerSession>() {
+          public int compare(RacePlayerSession o1, RacePlayerSession o2){
+            long t1 = resultsByPlayerId.containsKey(o1.getPlayerId()) ? resultsByPlayerId.get(o1.getPlayerId()).getRecord(laps) : Long.MAX_VALUE;
+            long t2 = resultsByPlayerId.containsKey(o2.getPlayerId()) ? resultsByPlayerId.get(o2.getPlayerId()).getRecord(laps) : Long.MAX_VALUE;
+            return Long.compare(t1, t2);
+        }}) ;

```

Comment:
I like to use lambda functions wherever possible because it minimizes the code a lot in the end. Would you want to write these as lambda functions?
```suggestion
        Collections.sort(startOrderSessions, (RacePlayerSession o1, RacePlayerSession o2) -> {
            long t1 = resultsByPlayerId.containsKey(o1.getPlayerId()) ? resultsByPlayerId.get(o1.getPlayerId()).getRecord(laps) : Long.MAX_VALUE;
            long t2 = resultsByPlayerId.containsKey(o2.getPlayerId()) ? resultsByPlayerId.get(o2.getPlayerId()).getRecord(laps) : Long.MAX_VALUE;
            return Long.compare(t1, t2);
        }) ;
```

Reply:
So, I don't typically use lambda functions mostly cause I learned to program before they really were a thing and I never remember the syntax. I can change it since it is essentially the same thing.